test/system/030-run.bats

```diff
@@ -818,9 +818,19 @@ EOF
 
 @test "podman run defaultenv" {
     run_podman run --rm $IMAGE printenv
-    assert "$output" =~ "TERM=xterm" "env includes TERM"
+    assert "$output" !~ "TERM=" "env doesn't include TERM by default"
     assert "$output" =~ "container=podman" "env includes container=podman"
 
+    run_podman 1 run -t=false --rm $IMAGE printenv TERM
+    assert "$output" == "" "env doesn't include TERM"
+
+    run_podman run -t=true --rm $IMAGE printenv TERM
+    output_cleaned=${output//[^a-zA-Z0-9_.]/} 
```

Comment:
This is the line with trailing whitespace. 

Hint: there are tools available for [vim](https://vim.fandom.com/wiki/Highlight_unwanted_spaces) and [emacs](https://www.emacswiki.org/emacs/DeletingWhitespace) that will highlight and nuke unwanted whitespace.

On reflection, this line is a little too magical. I'd like to see a comment such as `# -i uses CRLF terminators; strip the CR` or even this instead (removing the magic cleanup line):
```bash
        assert "$output" == $'xterm\r' "env includes default TERM"
```